src/commands/publish/preview/mod.rs

```diff
@@ -34,6 +34,7 @@ pub fn preview(
     body: Option<String>,
     livereload: bool,
     verbose: bool,
+    browser: bool,
```

Comment:
the flag is "terminal" but this attribute says browser... might be more maintainable if they are aligned- what do you think?

i sorta think that it might be nice to have the flag be browser instead of terminal... backwards compat has us a bit stuck though. gonna think on this- but would love to hear your thoughts!

Reply:
My preference would be that the default opens the browser and that the flag is browser. The reason I implemented it this way is because of the way things are today. That being said, I don't think we really need to worry too much about backwards compatibility wrt preview, but I could be wrong.

Reply:
should it be configurable via another method as well? there's the whole idea that if there's a flag for it, there should be a config setting for it.

Reply:
also, perhaps it isn't a no arg flag, but a bool value flag, like `--browser=true`, with a default